File: web/req-http.py
#!/usr/bin/env python3

import urllib.request

with urllib.request.urlopen('http://localhost:5000/test') as response:
    if response.status == 200:
        print('[200 OK]', response.getcode())
        print(response.read().decode('UTF-8'))
    else:
        print('[ERROR] status = ', response.status)

File: web/req-http.js
// URL指定(method省略時GET)
fetch('http://localhost:5000/test')
    .then((response) => {
        if (response.ok) {
            return response.text()
        } else {
            throw new Error()
        }
    })
    // レスポンスを正常にテキストに変換できた場合
    .then(text => console.log(text))
    // エラーだった場合
    .catch(error => console.error(error))